service/rcv_server.conf

```diff
@@ -7,12 +7,12 @@ is_front=True
 width=320
 height=240
 fps=30
-video_source=video:/home/bazulay/dev_robotics/rcv_videos/edited_videos/2019_02_26_19_39_30.mp4
+video_source=camera:-1
```

Comment:
why -1 ?

Reply:
-1 searches for the first available camera